package.nls.json

```diff
@@ -63,22 +63,22 @@
     "AWS.error.no.error.code": "No error code",
     "AWS.error.sam.local.package_json_not_found": "Unable to find package.json related to {0}",
     "AWS.error.check.logs": "Check the logs for more information by running the \"{0}\" command from the Command Palette.",
-    "AWS.explorerNode.addRegion": "Add a region to view functions...",
-    "AWS.explorerNode.addRegion.tooltip": "Click to add a region to view functions...",
+    "AWS.explorerNode.addRegion": "Add a region to AWS Explorer...",
+    "AWS.explorerNode.addRegion.tooltip": "Click here to add a region to AWS Explorer.",
     "AWS.explorerNode.lambda.noFunctions": "[No Functions found]",
     "AWS.explorerNode.cloudFormation.noFunctions": "[Stack has no Lambda Functions]",
     "AWS.explorerNode.cloudformation.noStacks": "[No Stacks found]",
     "AWS.explorerNode.cloudFormation.error": "Error loading CloudFormation resources",
     "AWS.explorerNode.container.noItems": "[no items]",
-    "AWS.explorerNode.lambda.retry": "Unable to load Lambda Functions, click here to retry",
+    "AWS.explorerNode.error.retry": "Unable to get child nodes, click here to retry",
     "AWS.explorerNode.lambda.error": "Error loading Lambda resources",
     "AWS.explorerNode.schemas.error": "Error loading Schemas resources",
     "AWS.explorerNode.schemas.noRegistry": "[No Schema Registries]",
     "AWS.explorerNode.registry.error": "Error loading registry schema items",
     "AWS.explorerNode.registry.noSchemas": "[no schemas in this registry]",
     "AWS.explorerNode.registry.registryName.Not.Found": "Registry name not found",
     "AWS.explorerNode.signIn": "Connect to AWS...",
-    "AWS.explorerNode.signIn.tooltip": "Connect to AWS using a credential profile",
+    "AWS.explorerNode.signIn.tooltip": "Click here to select credentials the AWS Toolkit will connect to AWS with",
```

Comment:
nit: Maybe something more like:

`Click here to select credentials for the AWS Toolkit`

The current tooltip is way too long and "select credentials the" is just weird sounding

Reply:
Agreed, will fix